src/Microsoft.OpenApi.Tool/OpenApiService.cs

```diff
@@ -21,6 +20,8 @@ static class OpenApiService
             FileInfo output,
             OpenApiSpecVersion version,
             OpenApiFormat format,
+            string filterByOperationIds,
```

Comment:
The name of the variable would subtly suggest that this holds a boolean value. Maybe rename it to `operationIds`?

Reply:
The parameters have to match the command option i.e if the command option is `--filterByTags`, then the param supplied in the command handler has to be `filterByTags`, otherwise it doesn't get mapped correctly while invoking the handler.
Same case for `operationIds`.